components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/JustInTimeProvisioningConfig.java

```diff
@@ -144,6 +146,21 @@ public boolean isAssociateLocalUserEnabled() {
         return associateLocalUserEnabled;
     }
 
+    /**
+     * To associate existing local user when JIT user provisioning is enabled.
+     *
+     * @param syncAttributeMethod to specify whether to associate existing local user when JIT user provisioning.
+     */
```

Comment:
The method comment implies a wrong message. This should be about attribute syncing.